blazor/diagram/connectors/segments/orthogonal.md

```diff
@@ -154,3 +154,83 @@ N> You need to mention the segment type as you mentioned in the connector type.
+
+The [Orthogonal](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Diagram.ConnectorSegmentType.html#Syncfusion_Blazor_Diagram_ConnectorSegmentType_Orthogonal) connector supports an arbitrary number of segments between its source and target points. By default, these  [Segments](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Diagram.Connector.html#Syncfusion_Blazor_Diagram_Connector_Segments)  are displayed with a circle shape in Grey color. You can customize the appearance of these segments using the SegmentThumbSettings property. This property allows you to choose from several predefined shapes. The shapes will be visible only when the DragSegmentThumb enum is added to the connector constraints. The following predefined shapes are provided:
+
+| Shape name | Shape |
+|-------- | -------- |
+|Rhombus| ![Rhombus](../../images/RhombusThumb.png) |
+| Square | ![Square](../../images/SquareThumb.png) |
+| Rectangle | ![Rectangle](../../images/RectangleThumb.png) |
+| Ellipse |![Ellipse](../../images/EllipseThumb.png) |
+| Circle |![Circle](../../images/CircleThumb.png) |
+|Arrow| ![Arrow](../../images/ArrowThumb.png) |
+| OpenArrow | ![OpenArrow](../../images/OpenArrowThumb.png) |
+| Fletch|![Fletch](../../images/FletchThumb.png) |
+|OpenFetch| ![OpenFetch](../../images/OpenFetchThumb.png) |
+| IndentedArrow | ![IndentedArrow](../../images/IndentedThumb.png) |
+| OutdentedArrow | ![OutdentedArrow](../../images/OutdentedThumb.png) |
+| DoubleArrow |![DoubleArrow](../../images/DoubleArrowThumb.png) |
+
+The following code example illustrates how to customize orthogonal segment thumb shape.
+
+```cshtml
+@using Syncfusion.Blazor.Diagram
+<SfDiagramComponent @ref="Diagram" Width="1000px"  Height="500px" Connectors="@connectors">
+</SfDiagramComponent>
+@code
+{
+    SfDiagramComponent Diagram;
+    DiagramObjectCollection<Connector> connectors = new DiagramObjectCollection<Connector>();
+    protected override void OnInitialized()
+    {
+        Connector Connector = new Connector()
+        {
+            ID = "Connector2",
+            Constraints = ConnectorConstraints.Default | ConnectorConstraints.DragSegmentThumb,
+            Type = ConnectorSegmentType.Orthogonal,
+            SourcePoint = new DiagramPoint { X = 400, Y = 100 },
+            TargetPoint = new DiagramPoint { X = 500, Y = 200 },
+            SegmentThumbSettings = new SegmentThumbSettings() {Shape = SegmentThumbShapes.IndentedArrow}
+        };
+        connectors.Add(Connector);
+    }
+}
+```
+You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/Blazor-Diagram-Examples/tree/master/UG-Samples/Connectors/Segments/OrthogonalThumbShape)
+
+![Editing Orthogonal Segment in Blazor Diagram](../../images/OrthogonalThumbSettings.png)
+
+When InheritSegmentThumbShape is enabled in the connector constraints, the shape specified in the diagram will be applied to the connector segment thumb.
+
```

Comment:
When the `InheritSegmentThumbShape` constraint is enabled in the connector, the shape specified at the diagram level will be applied to the connector segment thumbs. This allows for consistent segment thumb shapes across the entire diagram.